Documentation/xcassets/swift2.md

```diff
@@ -21,6 +21,8 @@ You can customize some elements of this template by overriding the following par
 | Parameter Name | Default Value | Description |
 | -------------- | ------------- | ----------- |
 | `enumName` | `Asset` | Allows you to change the name of the generated `enum` containing all assets. |
+| `colorTypeName` | `ColorAsset` | Allows you to change the name of the struct type representing a color. |
+| `imageTypeName` | `ImageAsset` | Allows you to change the name of the struct type representing an image. |
 | `noAllValues` | N/A | Setting this parameter will disable generation of the `allValues` constant. |
```

Comment:
Would be breaking I know, but I wonder if a sensible default wouldn't instead be to _not_ generate `allColors`/`allImages` (as that could generate a really huge array, and in most cases for nothing as I personally never saw the need for having such an array of all your images in your whole app anyway), and thus turning that parameter to `allValues` instead with the inverse meaning? Thoughts?